packages/sdk/src/HopBridge.ts

```diff
@@ -454,11 +454,30 @@ class HopBridge extends Base {
       estimatedReceived = estimatedReceived.sub(totalFee)
     }
 
+    let requiredLiquidity = hTokenAmount
+    let includePendingAmount = false
+    if (destinationChain.equals(Chain.Ethereum)) {
+      if (
+        sourceChain.equals(Chain.Optimism) ||
+        sourceChain.equals(Chain.Arbitrum)
+      ) {
+        includePendingAmount = true
+      }
+    }
+
+    if (includePendingAmount) {
+      const bridgeContract = await this.getBridgeContract(sourceChain)
+      const pendingAmount = await bridgeContract.pendingAmountForChainId(
+        destinationChain.chainId
+      )
+      requiredLiquidity = requiredLiquidity.add(pendingAmount).add(hTokenAmount) // account for transfer root bonds
```

Comment:
Same question about the `2x` as above.

Reply:
It's already being double counted because  `requiredLiquidity` is initialized as `hTokenAmount`